src/ru/geekbrains/homeworks/homework3/HomeWork3.java

```diff
@@ -0,0 +1,197 @@
+package ru.geekbrains.homeworks.homework3;
+
+import java.util.Arrays;
+
+public class HomeWork3 {
+    public static void main(String[] args) {
+
+        /**Задача 1
+         * Меняем 0 на 1 и наоборот.
+         * Здесь и далее вывод в консоль
+         * в конце задачи закоментирован т.к.
+         * не требуется согласно ДЗ и добавлен для удобства
+         * проверки преподавателя (наверное, но это не точно).
+         */
+
+        int[] arr = {1, 0, 0, 1};
+
+        {
+            for (int i = 0; i < arr.length; i++) {
+                if (arr[i] == 1) {
+                    arr[i] = 0;
+                } else arr[i] = 1;
+            }
+        }
+//        System.out.println(Arrays.toString(arr));
+
+
+        /**
+         * Задача 2
+         * Заполняем массив длинной в 100
+         * значениями от 1 до 100
+         */
+        int[] arr2 = new int[100];
+        {
+            for (int i = 0; i < arr2.length; i++) {
+                arr2[i] = i + 1;
+            }
+        }
+
+//         System.out.println(Arrays.toString(arr2));
+
+
+        /**
+         * Задача 3
+         * Умножаем значение массива на 2, если оно меньше 6
+         */
+        int[] arr3 = {1, 5, 3, 2, 11, 4, 5, 2, 4, 8, 9, 1};
+        {
+            for (int i = 0; i < arr3.length; i++) {
+                if (arr3[i] < 6) {
+                    arr3[i] *= 2;
+                }
+            }
+        }
+
+
+//         System.out.println(Arrays.toString(arr3));
+
+
+/**
+ * Задача 4
+ * Вышиваем крестиком )))
+ */
+        int[][] table = new int[5][5];
```

Comment:
А само заполнение массива я так и не увидел)  то есть если я после цикла выведу массив на экран, то там будут 0 везде

Reply:
```suggestion
        int[][] table = new int[5][5];
```
Скажите пожалуйста, согласно задания заполнять надо только диагональные элементы, поэтому остальные не заполнял. Чем необходимо заполнить другие элементы? Каким-то любым одним числом или по порядку числами от 1 до 25?

Reply:
А чем они будут заполнены по умолчанию? :) 

Я к тому, что ожидал увидеть arr[i][j] = 1 вместо system.out

В явном виде присвоения не написано.

Reply:
Понял, у меня был вывод в печать, а не заполнение массива. Заполнил массив, не диагонали заполнил 8ками.